edx_solutions_projects/apps.py

```diff
@@ -0,0 +1,15 @@
+"""
+app configuration
```

Comment:
It would be great to include some information on how this app configuration feature of Django works -- at least provide a link to the Django documentation?